graphs/bfs.py

```diff
@@ -31,13 +34,12 @@ def bfs(graph, start):
     >>> ''.join(sorted(bfs(G, 'A')))
     'ABCDEF'
```

Comment:
```suggestion
    'ABCDEFGHI'
```

Reply:
IMHO:
```
    >>> ''.join(bfs(G, 'A'))
    'ABECFIHGD'
```

Reply:
_I added a new graph, so the_ ~test~ _`function` needs to be fixed accordingly_ because currently it returns `ABCDEFGHI` not `ABECFIHGD`.  This is why Travis CI is failing this pull request.  Either the test must be fixed to match the function output or the function must be fixed to match the test's expected result.  They must agree for this pull request to be considered.

Reply:
Python _sets_ are defined to be [an unordered collection with no duplicate elements](https://docs.python.org/3/tutorial/datastructures.html#sets).  This means that they should not be used if the order of elements is important.